src/heads-up/components/app-tooltip.vue

```diff
@@ -1,152 +1,101 @@
-  }
+.tooltip[x-placement^="top"] .tooltip-arrow {
+  bottom: -5px;
+  margin: 0 5px;
+}
 
-  .vue-tooltip[x-placement^="bottom"] {
-    margin-top: 5px;
-  }
+.tooltip[x-placement^="bottom"] {
+  margin-top: 5px;
+}
 
-  .vue-tooltip[x-placement^="bottom"] .tooltip-arrow {
-    top: -10px;
-    border-width: 0 10px 10px 10px;
-    border-bottom-color: var(--toolbar-bg-color);
-    border-top-color: transparent !important;
-    border-left-color: transparent !important;
-    border-right-color: transparent !important;
-    margin-top: 0;
-    margin-bottom: 0;
-  }
+.tooltip[x-placement^="bottom"] .tooltip-arrow {
+  top: -5px;
+  margin: 0 5px;
+}
 
-  .-theme-with-dark-background .vue-tooltip[x-placement^="bottom"] .tooltip-arrow::after {
-    content: '';
-    display: block;
-    top: -7px;
-    border-width: 11px 11px 0 11px;
-    border-top-color: var(--base-color);
-    border-bottom-color: transparent !important;
-    border-left-color: transparent !important;
-    border-right-color: transparent !important;
-    margin-top: 0;
-    margin-bottom: 0;
-    width: 0;
-    height: 0;
-    border-style: solid;
-    position: absolute;
-    margin: 5px;
-    left: -16px;
-    z-index: -1;
-    transform: rotate(180deg);
-  }
+.tooltip[x-placement^="right"] {
+  margin-left: 5px;
+}
 
-  .vue-tooltip[x-placement^="top"] {
-    margin-bottom: 5px;
-  }
+.tooltip[x-placement^="right"] .tooltip-arrow {
+  left: -5px;
+  margin: 5px 0;
+}
 
-  .vue-tooltip[x-placement^="top"] .tooltip-arrow {
-    bottom: -10px;
-    border-width: 10px 10px 0 10px;
-    border-top-color: var(--toolbar-bg-color);
-    border-bottom-color: transparent !important;
-    border-left-color: transparent !important;
-    border-right-color: transparent !important;
-    margin-top: 0;
-    margin-bottom: 0;
-  }
+.tooltip[x-placement^="left"] {
+  margin-right: 5px;
+}
 
-  .-theme-with-dark-background .vue-tooltip[x-placement^="top"] .tooltip-arrow::after {
-    content: '';
-    display: block;
-    bottom: -7px;
-    border-width: 11px 11px 0 11px;
-    border-top-color: var(--base-color);
-    border-bottom-color: transparent !important;
-    border-left-color: transparent !important;
-    border-right-color: transparent !important;
-    margin-top: 0;
-    margin-bottom: 0;
-    width: 0;
-    height: 0;
-    border-style: solid;
-    position: absolute;
-    margin: 5px;
-    left: -16px;
-    z-index: -1;
-  }
+.tooltip[x-placement^="left"] .tooltip-arrow {
+  right: -5px;
+  margin: 5px 0;
+}
 
-  @media (min-width: 400px) {
-    .vue-tooltip {
-      width: 200px;
-    }
-  }
+.tooltip[aria-hidden="true"] {
+  visibility: hidden;
+  opacity: 0;
+  transition: opacity 0.15s, visibility 0.15s;
+}
 
-  @media (min-width: 500px) {
-    .vue-tooltip {
-      width: 300px;
-    }
-  }
+.tooltip[aria-hidden="false"] {
+  visibility: visible;
+  opacity: 1;
+  transition: opacity 0.15s;
+  box-shadow: 0px 0px 6px 1px rgba(0, 0, 0, 0.2);
+  width: 150px;
```

Comment:
Where does this value come from? I find the tooltip very narrow. instead I'd perfer a `max-width` of `75vw`.